components/automate-ui/src/app/modules/infra-proxy/infra-tab-change/infra-tab-change.component.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { InfraTabChangeComponent } from './infra-tab-change.component';
+
+describe('InfraTabChangeComponent', () => {
+  let component: InfraTabChangeComponent;
+  let fixture: ComponentFixture<InfraTabChangeComponent>;
+
+  beforeEach(async () => {
+    await TestBed.configureTestingModule({
+      declarations: [ InfraTabChangeComponent ]
+    })
+    .compileComponents();
+  });
+
+  beforeEach(() => {
+    fixture = TestBed.createComponent(InfraTabChangeComponent);
+    component = fixture.componentInstance;
+    fixture.detectChanges();
+  });
+
+  it('should create', () => {
```

Comment:
Please add a couple real tests, too.

Reply:
I have removed this file as it will not specify any to run test case as I already added a test case regarding tabs in Create spec file.